app/models/energy/ScheduleBase.py

```diff
@@ -0,0 +1,60 @@
+"""Schedule Base Schema"""
+from pydantic import BaseModel, Schema, validator, ValidationError, UrlStr, ConstrainedStr
+from typing import List, Union
+from enum import Enum
+from uuid import UUID, uuid4
+from datetime import datetime
+
+
+class UnitType (str, Enum):
+    dimensionless = 'Dimensionless'
+    temperature = 'Temperature'
+    delta_temperature = 'DeltaTemperature'
+    precipitation_rate = 'PrecipitationRate'
+    angle = 'Angle'
+    convection_coefficient = 'ConvectionCoefficient'
+    activity_level = 'ActivityLevel'
+    velocity =  'Velocity'
+    capacity =  'Capacity'
+    power =  'Power'
+    availability =  'Availability'
+    percent = 'Percent'
+    control = 'Control'
+    mode = 'Mode'
+
+class DateTime(BaseModel):
```

Comment:
This was going to move to a more generic location right?

Reply:
Maybe break into a separate Date and Time classes with DateTime being a combination?

Reply:
@macumber I created a `ScheduleBase` , the `ScheduleFile` and `ScheduleRuleset` reference this file.